che-admin-api/che-core-api-plugin/pom.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2012-2015 Codenvy, S.A.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Codenvy, S.A. - initial API and implementation
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>che-admin-api-parent</artifactId>
+        <groupId>org.eclipse.che.core</groupId>
+        <version>3.13.1-SNAPSHOT</version>
+    </parent>
+    <artifactId>che-core-api-plugin</artifactId>
+    <packaging>jar</packaging>
+    <name>Che Core :: Admin API :: Plugin</name>
+    <properties>
+        <dto-generator-out-directory>${project.build.directory}/generated-sources/dto/</dto-generator-out-directory>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-aether-provider</artifactId>
+            <version>3.3.1</version>
```

Comment:
As part of the Che Admin war file only packaged alongside Eclipse Che for now.
(It's not inside che.war as we need to be able to administrate it)